### DISCUSSION TITLE
LAC训练数据集在哪里可以下载呢

### DISCUSSION THREAD

**Discussion by adeagle** [PaddlePaddle/PaddleNLP#884] [Q&A] [answered]

**Comment by joey12300** [comment]
@adeagle 您好，有关2.0版本的LAC模型可以参考该链接：https://github.com/PaddlePaddle/PaddleNLP/tree/develop/examples/lexical_analysis

**Comment by adeagle** [comment]
这只是部分数据集吧 我想训练一个跟你们精度一样的模型  去哪里下载数据集呢

**Comment by joey12300** [comment, marked as answer]
这里只提供样例数据，完整的数据集是不开源的~

**Comment by adeagle** [comment]
好的 谢谢啊
